doc(pe): Document policy options (#601)
armory/docs: content/en/docs/plugin-guide/plugin-policy-engine/packages/spinnaker.http.authz/tasks/type.updateApplication.md

            },
            "description": "description2",
            "email": "dasdasd@trest.com",
            "instancePort": 80,
            "lastModifiedBy": "myUserName",
            "name": "aftest2",
            "permissions": {
              "EXECUTE": [
                "productmanagers"
              ],
              "READ": [
                "productmanagers"
              ],
              "WRITE": [
                "productmanagers"
              ]
            },
            "repoProjectKey": "project",
            "repoSlug": "name",
            "repoType": "github",
            "trafficGuards": [],
            "updateTs": "1621444448000",
            "user": "myUserName"
          },
          "type": "updateApplication",
          "user": "myUserName"
        }
      ]
    },
    "method": "POST",
    "path": [
      "tasks"
    ],
    "user": {
      "isAdmin": false,
      "roles": [
        {
          "name": "armory-io",
          "source": "GITHUB_TEAMS"
        },
        {
          "name": "productmanagers",
          "source": "GITHUB_TEAMS"
        }
      ],
      "username": "myUserName"
    }
  }
}
```
</details>

## Example Policy

> This policy will disable the ability to create new applications, or update existing applications unless the applications have specified at least 1 role with 'write' permissions.

Reviewer: ```suggestion
This policy disables the ability to create new applications or update existing applications unless the applications have specified at least 1 role with 'write' permissions.
```

> **Note:** The Spinnaker UI is not currently able to display an error message when this policy denies the action.

Reviewer: ```suggestion
> **Note:** The UI does not display an error message when this policy denies the action.
```
